includes/functions/content-sync.php

```diff
@@ -198,7 +198,11 @@ function get_post_data( $post ) {
 			$canonical_url = $yoast_canonical;
 		}
 
-		$keywords = get_post_meta( $post->ID, '_yoast_wpseo_focuskw', true );
+		$yoast_focuskw = get_post_meta( $post->ID, '_yoast_wpseo_focuskw', true );
+		if ( ! empty( $yoast_focuskw ) ) {
+			// Limit focus keyphrase to max length of 128.
+			$keywords = [ substr( $yoast_focuskw, 0, 128 ) ];
```

Comment:
Although we probably won't be sending anything over 100 strings as the array content, do we want to go ahead and implement that as well? Per requirement: 

`steps to fix: allow an array of strings with min 0 and max 100 strings.`

Reply:
Yes, 
Thank you

Reply:
@oscarssanchez `$yoast_focuskw` will be a single string. So, `$keywords` will be an array with 1 length always. However, the max length of the string can be up to 128, which I already implemented. Please let me know If I missed or misunderstood anything here. 
Thanks.

Reply:
Sonds good to me. 
How will this be populated if they are not using yoast?

Reply:
@YMufleh it will be only populated if Yoast is installed. If they are not using Yoast, `keywords` field will be not sent in the request. (note: `keywords` field is optional)

Reply:
Noting here that WordPress, by default, does not provide `keywords` functionality and requires a plugin, like Yoast, to add on this SEO feature.  There are some other SEO plugins, but by default the plugin only supports Yoast.  If there are others that are determined to be in-use by other partners, then support for those SEO plugins can be added.

Reply:
@iamdharmesh , yes you are correct at this point we should not have more than 1 string, I was thinking more about future proofing, so something like `if (count($array) > 100) limit array` , but seems per @YMufleh we can leave this for now

Reply:
> l be only populated if Yoast is installed. If they are not using Yoast, `keywords` field will be n

Perhaps we could think using tags or something like that if no SEO plugin is instlled @jeffpaul @YMufleh ? I'm not sure what effect would that have in Sophi though.

Reply:
I believe tags are sent across in a different field/variable, right?  Just like categories?

Reply:
See categories: https://github.com/globeandmail/sophi-for-wordpress/blob/47fbb181999890297e3cad5594832c0443d5dd14/includes/functions/content-sync.php#L220
and tags: https://github.com/globeandmail/sophi-for-wordpress/blob/47fbb181999890297e3cad5594832c0443d5dd14/includes/functions/content-sync.php#L222

...right?